project_planning/project_tasks_planning_view.xml

```diff
@@ -0,0 +1,146 @@
+        <record id="view_task_planning_form" model="ir.ui.view">
+            <field name="name">project.task.planning.form</field>
+            <field name="model">project.task</field>
+            <field name="priority">2</field>
+            <field name="arch" type="xml">
+                <form string="Project" delete="false">
+                    <header>
+                        <field name="stage_id" widget="statusbar" clickable="True" options="{'fold_field': 'fold'}" modifiers="{}"/>
+                    </header>
+                    <sheet string="Task">
+                        <div class="oe_right">
+                            <field name="kanban_state" class="oe_inline" widget="kanban_state_selection" modifiers="{}"/>
+                        </div>
+                        <h1>
+                            <field name="id" modifiers="{'readonly': true}"/>
+                            <field name="name" placeholder="Task summary..." class="oe_inline" modifiers="{'required': true}"/>
+                        </h1>
+                        <group>
+                            <group>
+                                <field name="type" modifiers="{'required': true}"/>
+                                <field name="wbs" modifiers="{}"/>
+                                <field name="project_id" domain="[('state','not in', ('close', 'cancelled'))]" on_change="onchange_project(project_id)" context="{'default_use_tasks':1}" modifiers="{'required': true}"/>
+                            </group>
+                            <group>
+                                <field name="date_deadline" modifiers="{}"/>
+                                <field name="planned_hours" widget="float" on_change="onchange_planned(planned_hours, effective_hours)" modifiers="{}"/>
+                                <field name="priority" widget="priority" modifiers="{}"/>
+                            </group>
+                        </group>
+                            <group>
+                                <group class="oe_subtotal_footer oe_right" name="project_hours">
+                                    <field name="effective_hours" widget="float" modifiers="{'readonly': true}"/>
+                                    <label for="remaining_hours" string="Remaining"/>
+                                    <div>
+                                        <field name="remaining_hours" widget="float" modifiers="{}"/>
+                                    </div>
+                                    <field name="total_hours" widget="float" class="oe_subtotal_footer_separator" modifiers="{'readonly': true}"/>
+                                </group>
+                            </group>
+                            <field name="work_ids" modifiers="{}"/>
+                        <div class="oe_clear"/>
+                    </sheet>
+                </form>
+            </field>
+        </record>
+        <record id="view_task_planning_tree" model="ir.ui.view">
```

Comment:
It would be better to have this view editable tree and add a link so that we can edit the task in form view